cypress/e2e/hide-question-numbers-interactives.ts

```diff
@@ -0,0 +1,149 @@
+import ActivityPage from "../support/elements/activity-page";
+
+const activityPage = new ActivityPage;
+
+context("Activity hide question numbers checked", () => {
+  before(() => {
+    cy.visit("?activity=sample-activity-hide-question-number");
+    activityPage.getActivityTitle().should("contain", "Test Hide Question Number Setting");
+  });
+  describe("Hide question numbers checked in activity",() => {
+    it.only("Verify the left column has the hide question number setting OFF and the right has it ON.",()=>{
```

Comment:
FYI: after an incident like this in CODAP, we added `eslint-plugin-mocha` and enabled the `no-exclusive-tests` rule so that this kind of error can't slip through.

Reply:
Ah, yes, that `only` was a mistake. I do like the way we do it in CODAP with the `es-lint-plugin-mocha` and `no-exclusive-tests` rule.